src/routes/uniswap.route.js

```diff
@@ -53,25 +51,27 @@ router.post('/sell-price', async (req, res) => {
   const paramData = getParamData(req.body)
   const baseTokenAddress = paramData.base
   const quoteTokenAddress = paramData.quote
+  const amount = paramData.amount
 
   try {
     // fetch the optimal pool mix from uniswap
-    const route = await uniswap.fetch_route(
+    const { trade, expectedOut} = await uniswap.priceSwapIn(
       baseTokenAddress,     // tokenIn is base asset
       quoteTokenAddress,    // tokenOut is quote asset
+      amount
     )
 
-    if (route != null) {
+    if (trade != null && expectedOut != null) {
       res.status(200).json({
         network: uniswap.network,
         timestamp: initTime,
         latency: latency(initTime, Date.now()),
         base: baseTokenAddress,
         quote: quoteTokenAddress,
-        amount: parseFloat(paramData.amount),
-        expectedOut: parseFloat(paramData.amount * route.midPrice.toSignificant(8)),
-        price: route.midPrice.toSignificant(8),
-        swaps: route.path,
+        amount: amount,
+        expectedOut: expectedOut.toSignificant(8),
+        price: trade.executionPrice.toSignificant(8),
+        swaps: trade,
```

Comment:
Rename the output variable to `trade` or `route`, since `swaps` is specific to Balancer